tests/pipelines/cogvideox/test_cogvideox.py

```diff
@@ -148,8 +148,8 @@ def test_inference(self):
         video = pipe(**inputs).frames
         generated_video = video[0]
 
-        self.assertEqual(generated_video.shape, (9, 3, 16, 16))
-        expected_video = torch.randn(9, 3, 16, 16)
+        self.assertEqual(generated_video.shape, (8, 3, 16, 16))
```

Comment:
We need to change this because now we don't implicity do `num_frames += 1` in the pipeline. This is the more correct thing to do. In the pipeline, I've defaulted to `num_frames=49` (which is the expected default for CogVideoX).